### DISCUSSION TITLE
502 Bad Gateway

### DISCUSSION THREAD

**Discussion by asamohammed** [community/community#44169] [GitHub Education]
Hi, 
I applied for the student pack 1 month ago and it still has not been approved or denied.
I have not received any updates, every time I go to the website to check on my application I get a 502 Bad Gateway error.

Is this just me or is this happening to everyone else?

<img width="310" alt="badgateway" src="https://user-images.githubusercontent.com/106772665/212217198-ce984c4c-4d53-448f-834b-618f8dc6ce75.png">

**Comment by pat-gc** [comment]
are you sure it hasnt been accepted i thought i was in the same situation try and apply for a free jetbrains license or something with github 
![image](https://user-images.githubusercontent.com/97562509/212497926-b4c97091-aea7-44b1-a589-44153c48e225.png)
i also didnt receive anything so i just waited till today i was tired of waiting so i just tried it because whats the worst that can happen you know.
i hope that works for you like it did for me
however i have the same 502 error when i try to go to any page that starts with https://education.github.com
![image](https://user-images.githubusercontent.com/97562509/212497987-a06e2f63-fd2e-4c30-a5d0-ddb2f7b8a2ea.png)
i also need help for that

**Comment by pat-gc** [comment]
aperently its a problem for a lot of ppl and they are trying to solve it look for error 502 and you will see a bunch of discussions about it

**Comment by pat-gc** [comment]
theres a discussion with answers here : https://github.com/orgs/community/discussions/41095
